Modules/Loadable/SubjectHierarchy/Widgets/qMRMLSubjectHierarchyTreeView.cxx

```diff
@@ -771,6 +771,10 @@ void qMRMLSubjectHierarchyTreeView::setCurrentNode(vtkMRMLNode* node)
   {
     return;
   }
+  if (node->GetHideFromEditors())
```

Comment:
Could we add a comment here and in the location where the subject hierarchy node are created to have developed reading this code or the code responsible to create the node understand implication of changing the logic ?

Reply:
That would be here:

https://github.com/Slicer/Slicer/blob/c93da36af30dee43baf2140b1eb4b865af386cf7/Modules/Loadable/SubjectHierarchy/Widgets/qSlicerSubjectHierarchyPluginLogic.cxx#L241-L249